docs/backtracking/n-queens.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: N Queens
+slug: /backtracking/n-queens
+---
+
+## Introduction
+
+The N Queen problem is placing N chess queens on an N x N chessboard such that no two queens can attack each other.
+The chess queens can attack in any of the following directions: horizontal, vertical, and diagonal.
+Output is the N x N binary matrices showing possible positions for queens such that they cannot attack each other, where 1 represents the correct place for the queen. If no such valid solution exists, then output is `false`.
```

Comment:
Looks too similar to [this](https://www.geeksforgeeks.org/n-queen-problem-backtracking-3/). Kindly don't plagiarize.